text/0000-trait_stable_vtable.md

```diff
@@ -108,7 +108,7 @@ struct VTable{
     size: usize,
     align: usize,
     drop_in_place: Option<unsafe extern"C" fn(*mut ())->()>,
-    reserved: *mut ()
+    dealloc:Option<unsafe extern"C" fn(*mut ())->()>,
```

Comment:
The type is for *exposition-only*. The type does not actually exist in the RFC, as part of the core language or standard library. If I actually defined the type, I'd definately put something like `enum Empty{}` in the parameter list to prevent it being called. The intent is simply to express the layout of the type.